### PR TITLE
UpdateChangeLog & Version for rel-1.1

### PR DESCRIPTION
**Pull request by yashmeet29**
UpdateChangeLog & Version for rel-1.1

### CODE REVIEW

**Review by rishikunnath2747 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Version 1.1.0
+
+### Changed
+Create, Read, Rename and Delete of attachments are not supported if repository is versioned.
```
Comment (dheerasameerk) [inline CHANGELOG.md]: It should be added not changed.
The message could be that "Added additional validation to check if repository is versioned and show UI message that versioned repository is not supported".

Comment (yashmeet29) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Version 1.1.0
+
+### Added
+Added additional validation to check if repository is versioned and show UI message that versioned repository is not supported".
```
Comment (dheerasameerk) [inline CHANGELOG.md]: Remove " at the end. Remove the word UI.

Comment (yashmeet29) [reply CHANGELOG.md]: Done

**Review by dheerasameerk — APPROVED** [review]

**Pull request merged**